src/content/en/updates/2017/01/payment-request-updates.md

```diff
@@ -0,0 +1,139 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: In Chrome 56 and 57, there are a few changes made to the Payment Request API following the spec changes. Learn what they are and make changes to your own implementations.
+
+{# wf_updated_on: 2017-01-31 #}
+{# wf_published_on: 2017-01-31 #}
+{# wf_tags: paymentrequest,chrome56,chrome57 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: In Chrome 56 and 57, there are quite a few changes made to the Payment Request API following the spec changes. Learn what they are and make changes to your own implementation. #}
+
+Since launch of the Payment Request API in Chrome 53, there are a few changes
+made to the API. These changes won't break the functionalities of your working
+code, but we will never. We recommend you to add [a
+shim](https://storage.googleapis.com/prshim/v1/payment-shim.js) to your code so
+that future changes won't break your product.
+
+Note: All changes described here are already reflected in [the existing
+integration guides](https://developers.google.com/web/fundamentals/discovery-and-monetization/payment-request/). You can
+
+## Chrome 57
+### PaymentRequest is now available inside iframe
+Payment Request API can now be called from within an `iframe` by adding the
+`allowpaymentrequest` attribute to the `iframe` element.
+
+    <iframe src="URL_INCLUDING_PAYMENT_REQUEST_CALL" allowpaymentrequest></iframe>
+
+### PaymentMethodData supports "basic-card"
+The first argument to `PaymentRequest` initialization involves an array of
+payment method data. The `PaymentMethodData` format has been altered in this
+release.
+
+**Old:**
+
+    var methodData = [{
+     supportedMethods: ['visa', 'mastercard', 'amex', 'jcb']
+    }];
+    var request = new PaymentRequest(methodData, details, options);
+
+**New:**
```

Comment:
Please use the comparison styles - see https://developers.google.com/web/resources/style-guide#comparisons